doctrine/doctrine-migrations-bundle/3.1/config/packages/doctrine_migrations.yaml

```diff
@@ -3,4 +3,4 @@ doctrine_migrations:
         # namespace is arbitrary but should be different from App\Migrations
         # as migrations classes should NOT be autoloaded
         'DoctrineMigrations': '%kernel.project_dir%/migrations'
-    enable_profiler: '%kernel.debug%'
+    enable_profiler: false # '%kernel.debug%' 
```

Comment:
The comment should be removd entirely IMO.

Reply:
Agreed